core/test/financial_templates/Liquidatable.js

```diff
@@ -256,7 +292,18 @@ contract("Liquidatable", function(accounts) {
         assert.equal(liquidation.disputer, disputer);
         assert.equal(liquidation.disputeTime.toString(), disputeTime.toString());
       });
-      it("Liquidation has already been disputed", async () => {
+      it("Dispute initiates an oracle call", async () => {
+        const disputeTime = await liquidationContract.getCurrentTime();
+        await liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer });
+        // Oracle should have an enqueued price after calling dispute
+        const pendingRequests = await mockOracle.getPendingQueries();
+        assert.equal(
+          pendingRequests[0]["identifier"],
+          priceTrackingIdentifier.padEnd(pendingRequests[0]["identifier"].length, 0)
+        );
+        assert.equal(pendingRequests[0]["time"], disputeTime);
```

Comment:
tiny style nit: prefer `.` syntax for accessing js objects:
```
pendingRequests[0].time
```

Here and elsewhere.

Reply:
noted, will update